app/scripts/views/components/pieChart/pieChart.js

```diff
@@ -100,6 +108,18 @@
           .ordering(function(d) {
             return d.key;
           });
+        v.chart.on("postRender",function(){
+          initLabels();
+          initReactData();
+        });
+        v.chart.on("preRedraw",function(){
+          removeMarker();
+        });
+        v.chart.on("postRedraw",function(){
+          if ( $("#"+v.opts.charDivId).length ) {
+            updatePieLabels();
+          }
+        });
```

Comment:
This functionality is there at https://github.com/cBioPortal/iViz/pull/40/files#diff-54aac811532d56db419d4d021f8490beL141